code/modules/cargo/packs/food.dm

```diff
@@ -245,3 +245,16 @@
 					/obj/item/melee/flyswatter)
 	crate_name = "beekeeping starter crate"
 	crate_type = /obj/structure/closet/crate/hydroponics
+
+/datum/supply_pack/food/ration
+	name = "Ration Crate"
+	desc = "6 standerd issue rations."
+	cost = 3000
```

Comment:
3000 is a bit expensive. For context, you could buy 3 winchester rifles for 2000

Reply:
I'm comparing it to pizza which is 6000. I think that's a greater issue that most of the crates are overpriced or the guns are too cheap

        name = "Donk Pocket Variety Crate"
	desc = "Featuring a line up of Donk Co.'s most popular pastry!"
	cost = 2000

	name = "Pizza Crate"
	desc = "Best prices on this side of the galaxy. All deliveries are guaranteed to be 99% anomaly-free!"
	cost = 6000 // Best prices this side of the galaxy.

Reply:
yeah food doesn't need to be that expensive.

Reply:
name = "Cat Crate"
	desc = "The cat goes meow! Comes with a collar and a nice cat toy! Cheeseburger not included."//i can't believe im making this reference
	cost = 5000 //Cats are worth as much as corgis.

Reply:
you said the issue is that crates are overpriced. You don't need to contribute to the issue.

Reply:
dropping it to 2k will fix some of the other prices in a later pr

Reply:
I would say 600-1200 myself.

Reply:
I would say thats low balling it since the rations are ALOT of food